vochain/state/processblockregistry.go

```diff
@@ -52,7 +53,7 @@ func (pbr *ProcessBlockRegistry) MinStartBlock(fromBlock uint32) (uint32, error)
 func (pbr *ProcessBlockRegistry) MaxEndBlock(fromBlock uint32) (uint32, error) {
 	maxEndBlock := fromBlock
 	if err := pbr.db.Iterate(pbrDBPrefix, func(pid, _ []byte) bool {
-		p, err := pbr.state.Process(pid, false)
+		p, err := pbr.state.Process(bytes.Clone(pid), false)
```

Comment:
AFAIU this is not REALLY needed, but it also doesn't hurt to be extra-safe, especially if Process method changes in the future.

Reply:
Umm, the Clone is not necessary since this is a callback function and has its own context. Am I wrong @mvdan ?